### PR TITLE
Add individual volume set options from group-samba

### PR DESCRIPTION
**Pull request by anoopcs9**
Since we do not have a configuration in _gluster-ansible-cluster_ to do _group volume set_ on volumes, we have to set each volume option from `group-samba` profile.

### CODE REVIEW

**Comment by anoopcs9** [comment]
> Is it just that this is the only way, gluster-ansible is able to take additional options to set?

At least that's what I have seen somewhere else which consumes _gluster-ansible_.
I have asked for clarification from concerned team.

File: vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml [hunk 1]
```diff
@@ -37,9 +37,17 @@
       name: "{{ gluster_cluster_volume }}"
       options:
         {
-          server.allow-insecure: 'on',
```
Comment (obnoxxx) [inline vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml:40]: why is this removed?

File: vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml [hunk 2]
```diff
@@ -37,9 +37,17 @@
...
+          features.cache-invalidation: 'on',
+          features.cache-invalidation-timeout: '600',
           performance.cache-samba-metadata: 'on',
-          storage.batch-fsync-delay-usec: '0',
```
Comment (obnoxxx) [inline vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml:42]: why is this removed?

File: vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml (hunk 1 above)
Comment (anoopcs9) [reply vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml:40]: It is the default now.
```
Option: server.allow-insecure
Default Value: on
Description: (null)
```

File: vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml (hunk 2 above)
Comment (anoopcs9) [reply vagrant/ansible/roles/samba-glusterfs.setup/tasks/main.yml:42]: It is the default now:
```
Option: storage.batch-fsync-delay-usec
Default Value: 0
Description: Num of usecs to wait for aggregating fsync requests
```
